### DISCUSSION TITLE
SAE unter Bookworm 64Bit

### DISCUSSION THREAD

**Discussion by Dachmelker** [camueller/SmartApplianceEnabler#605] [Installation und Betrieb]
Guten Morgen,

Ich habe Probleme mit der Installation von SAE auf einem Raspberry mit der aktuellsten Version.
Bookworm 64 Bit
SAE will nicht starten

**Comment by camueller** [comment]
Ich habe vor einigen Wochen den SAE mit Bookworm erfolgreich installiert. Dafür hatte ich im Januar extra eine neues Image releast https://github.com/camueller/RaspiOSImageAutorunUSBShellScript/releases/tag/2023-12-11-lite

**Comment by Dachmelker** [comment]
funktioniert die Standard Installation über das Skript denn trotzdem?

oder muss ich da jetzt irgendwie anders rangehen?

https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Installation_DE.md

**Comment by camueller** [comment]
Sollte funktionieren, habe ich ja auch so gemacht. 

Ansonsten siehe https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Installation_DE.md#fehler-bei-der-installation
